README.md

```diff
@@ -64,9 +64,38 @@ Run it as
 See `jsonrcpproxy.py --help` for more options.
 
 
+## geth
```

Comment:
Perhaps we should add a explanation for Geth+Hera vs Geth+Wagon to give users more context.

Reply:
Please rename this to `geth + Wagon`